src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -453,6 +453,37 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->registerForAutoconfiguration(RouteLoaderInterface::class)
             ->addTag('routing.route_loader');
+
+        $this->addClassesToPreload([
+            'Psr\Log\LogLevel',
+            'Symfony\Component\Cache\Adapter\ApcuAdapter',
+            'Symfony\Component\Cache\Adapter\ArrayAdapter',
+            'Symfony\Component\Cache\Adapter\PhpArrayAdapter',
+            'Symfony\Component\Cache\Adapter\PhpFilesAdapter',
+            'Symfony\Component\Cache\CacheItem',
+            'Symfony\Component\DependencyInjection\Argument\RewindableGenerator',
+            'Symfony\Component\DependencyInjection\Argument\ServiceLocator',
+            'Symfony\Component\DependencyInjection\ContainerAwareInterface',
+            'Symfony\Component\DependencyInjection\ContainerAwareTrait',
+            'Symfony\Component\Dotenv\Dotenv',
+            'Symfony\Component\ErrorHandler\ErrorHandler',
+            'Symfony\Component\EventDispatcher\LegacyEventDispatcherProxy',
+            'Symfony\Component\HttpFoundation\AcceptHeader',
+            'Symfony\Component\HttpFoundation\AcceptHeaderItem',
+            'Symfony\Component\HttpFoundation\FileBag',
+            'Symfony\Component\HttpFoundation\HeaderBag',
+            'Symfony\Component\HttpFoundation\HeaderUtils',
+            'Symfony\Component\HttpFoundation\ParameterBag',
+            'Symfony\Component\HttpFoundation\ResponseHeaderBag',
+            'Symfony\Component\HttpFoundation\ServerBag',
```

Comment:
what about Request and Response ?

Reply:
the parameter and return types of public methods are preloaded too, so it's not needed:
https://github.com/symfony/symfony/blob/51a2cb6e3c180e48d042b8ff1e85781910552de7/src/Symfony/Component/DependencyInjection/Dumper/Preloader.php#L77-L95